app/views/editorial/editorial/index.html.haml

```diff
@@ -2,13 +2,16 @@
 
 %h1 Editorial
 
+= link_to('News', editorial_news_articles_path)
+
 - if @sections.count == 0
   %p.callout
     You are not registered as a collaborator on any sections.
 - else
   %ul
     - @sections.each do |section|
-      %li
-        = link_to(section.name, editorial_section_path(section))
-        - if can? :create_in, section
-          = link_to('Create new page', prepare_editorial_section_nodes_path(section))
+      - if section.name != 'news'
```

Comment:
Rather than filtering in the view, why not suppress that section in the controller? From the looks of this view, you don't need in in the `@sections` collection variable, do you?